### DISCUSSION TITLE
@quasar/qpdfviewer

### DISCUSSION THREAD

**Discussion by nachoalcazar** [quasarframework/quasar#11844] [Quasar App Extensions] [answered]
Hello,

I install @quasar/qpdfviewer and logs says me:

`quasar ext add @quasar/qpdfviewer`

`"Extension(@quasar/qpdfviewer): is not compatible with @quasar/app v3.2.6. Required version: ^1.0.0 || ^2.0.0"`

Anybody working in it? I can wait some months. 

I test some alternatives  but doesn't render very good in mobile platforms. Do you know any good library for vue3? The quasar extension in the pdfjs tests painted perfect the pdf with its tool buttons

Thanks

**Comment by nachoalcazar** [comment, marked as answer]
Excuse me:

Works OK with this installation:

```
quasar ext add @quasar/qpdfviewer@next
quasar ext remove @quasar/qpdfviewer@next
```

**Reply by hawkeye64** [reply]
You don't have to add `@next` for removal. For adding, that's just the tag to use (if it has been tagged). That information is not available after installation (probably ignored)
